### PR TITLE
Fabo/upgrade to stargate-4

### PR DESCRIPTION
**Pull request by faboweb**
first step towards IBC

### CODE REVIEW

File: network.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ export default {
     'Cosmos is a decentralized network of independent parallel blockchains, each powered by BFT consensus algorithms like Tendermint consensus.',
   logo: `logo.svg`,
   website: 'https://cosmos.network',
-  apiURL: 'https://lcd.nylira.net',
-  stakingDenom: 'ATOM',
+  apiURL: 'https://api.allorigins.win/raw?url=http://34.123.30.100:1317',
```
Comment (faboweb) [inline network.js:8]: @Bitcoinera see handling for CORS

Comment (Bitcoinera) [reply network.js:8]: oh, so easy? I was changing nuxt config like this:

```
  axios: {
    progress: false,
    retry: { retries: 3 },
    proxy: true,
  },

  proxy: {
    '/': 'https://cors-anywhere.herokuapp.com/',
  },
```

But it wasn't working :shrug: Although according to the documentation this also should work

**Comment by Bitcoinera** [comment]
It is actually `stargate-4`, isn't it?

image

**Comment by Bitcoinera** [comment]
image

**Comment by faboweb** [comment]
> It is actually `stargate-4`, isn't it?
> 
> image
> 

Oh. Will switch to stargate-5

**Comment by faboweb** [comment]
> Now I get this
> 
> 
> image
> 

For which signing methods?

**Comment by Bitcoinera** [comment]
Let's try now with the real `startgate-5`

**Comment by Bitcoinera** [comment]
Mmph, I'm getting `ECONNREFUSED`

image

File: apis/cosmos-reducers.js [hunk 2]
```diff
@@ -589,22 +594,22 @@ export function transactionReducer(transaction) {
...
+          message,
           transaction
         ),
-        rawTransaction: transaction,
         timestamp: transaction.timestamp,
-        memo: transaction.tx.value.memo,
+        memo: transaction.tx.body.memo,
         fees,
-        success: setTransactionSuccess(transaction, messageIndex),
+        success: setTransactionSuccess(transaction, messageIndex, network.id),
```
Comment (faboweb) [inline apis/cosmos-reducers.js:612]:
```suggestion
        success: setTransactionSuccess(transaction, messageIndex),
```

File: signing/transaction-manager.js [hunk 3]
```diff
@@ -56,21 +57,40 @@ export async function createSignBroadcast({
...
+    {
+      method: 'POST',
+      body: JSON.stringify(broadcastBody),
+    }
+  ).then((res) => res.json())
   assertIsBroadcastTxSuccess(broadcastResult)
 
+  if (
+    !assertIsBroadcastTxSuccess(broadcastResult) &&
+    broadcastResult.contents
```
Comment (faboweb) [inline signing/transaction-manager.js:87]: What is assertion fails and content is not available? I guess this check needs to move inside the if block

**Pull request merged**